src/coop/rchain/lang/folding/RholangFoldingBuilder.kt

```diff
@@ -0,0 +1,67 @@
+package coop.rchain.lang.folding
```

Comment:
In the project I included `.editorconfig` file which helps to keep formatting consistent. It is set to indent of 2 spaces. If you install EditorConfig plugin you can just reformat the code.

Reply:
Yes, I will install this plugin, and get all code formatted.